docs/src/concepts.md

```diff
@@ -0,0 +1,123 @@
+# Concepts
+
+This package enhances existing concepts from Sublime Text and introduces new concepts not native to Sublime Text. This page provides an overview of the most important capabilities. The capabilities/concepts are accessible in different ways. Some are accessible via the Context Menu by right-clicking with your mouse, or via the top Menu Bar. Others are part of an existing workflow. Almost all capabilities can also be bound to a key with a key binding.
+
+## Auto Complete [cooperating]
+
+The auto-complete functionality of Sublime Text is well-known to any user. It provides word completions from the current buffer, and, since ST4, completions from other files. It presents the auto-complete widget in a synchronous fashion.
+
+The LSP package enhances the auto-complete functionality.
+
+## Goto Definition [replacing]
+
+Sublime Text provides a "goto-def" feature by indexing the files in your project, and building a database out of the parsed files. The feature is accessible by clicking on Goto > Goto Definition. Sublime will attempt to jump to the definition of the word enclosing the caret. The files are parsed according to the `.sublime-syntax` associated to them. Entities which are assigned the `entity.name.*` scope are considered to be a "definition". Because a single `.sublime-syntax` file has no knowledge of the project structure, there may be multiple such "definitions".
```

Comment:
Why use the word `goto-def`? Nowhere in ST it's called using such abbreviation. Could use full words.